content/blog/coredns-in-kubernetes-via-kubeadm.md

```diff
@@ -0,0 +1,267 @@
++++
+title = "Deploying Kubernetes with CoreDNS using kubeadm"
+description = "A guide to installing CoreDNS in Kubernetes via kubeadm"
+tags = ["Kubernetes", "Service", "Discovery", "Kube-DNS", "Kubeadm", "DNS", "Documentation"]
+author = "sandeep"
++++
+
+Kubernetes 1.9 has recently been launched and it ships with CoreDNS being a part of it. 
+We can now install CoreDNS as the default service discovery via Kubeadm, which is the toolkit to install Kubernetes easily in a single step.
+
+Currently, [CoreDNS is Alpha in Kubernetes 1.9](https://github.com/kubernetes/features/issues/427). We have a roadmap which will make CoreDNS Beta in version 1.10 and eventually be the default DNS, replacing kube-dns.
+
+## Installing CoreDNS in fresh Kubernetes cluster
+In order to install CoreDNS instead of kube-dns for a fresh Kubernetes cluster, we need to use the `feature-gates` flag and set it to `true`. 
+Use the following command to install CoreDNS as default while installing a fresh Kubernetes cluster.
+~~~ text
+# kubeadm init --feature-gates CoreDNS=true
```

Comment:
Can you isolate the command line, and make it bold to show that is the important command.